docs(fix): add troubleshooting for halyard AccessDenied

aimeeu opened the pull request:
Add what to do when the Halyard container cannot access the shared
drives on the host machine.

Move section about kubectl/halyard installation order when not using
the Halyard container to the Assumptions section.

Fix code snippet indentation in lists.

Fix nginx-ingress name mismatch. https://armory.atlassian.net/browse/DOCS-74

Fixes: Jira-74

Signed-off-by: Aimee Ukasick <aimeeu.opensource@gmail.com>

brian-armory commented on _spinnaker_install_admin_guides/install-on-aws.md:
```diff
@@ -13,107 +13,121 @@ redirect_from:
-* You have the `kubectl` (Kubernetes CLI tool) installed and are able to use it to interact with your Kubernetes cluster
-* You have a persistent working directory in which to work in.  One option here is `~/aws-spinnaker`
-* You will create AWS resources, such as service accounts, that will be permanently associated with your Spinnaker cluster
+- If using EKS, you can use the `aws` CLI tool to interact with the AWS API, and configure / communicate with the following:
```
no comma after API

brian-armory commented on _spinnaker_install_admin_guides/install-on-aws.md:
```diff
@@ -123,156 +137,163 @@ Then, copy your kubeconfig file (this is typically located in `~/.kube/config) t
-   SPINNAKER_SERVICE_ACCOUNT_NAME=spinnaker-service-account
-
-   ./spinnaker-tools create-service-account \
-     --kubeconfig ${SOURCE_KUBECONFIG} \
```
Edit: Looks like this is broken on live too. https://share.getcloudapp.com/nOumbADW

Need 4 spaces so that markdown understands its part of that one ordered list item. It currently has 3 spaces, which is why it's broken. Will look like this once it's fixed: 
https://share.getcloudapp.com/4guxekel

brian-armory commented on _spinnaker_install_admin_guides/install-on-aws.md:
```diff
@@ -123,156 +137,163 @@ Then, copy your kubeconfig file (this is typically located in `~/.kube/config) t
 You should be left with a file called `kubeconfig-spinnaker-system-sa` (or something similar, if you're using a different namespace for spinnaker)
 
-## Create the S3 Bucket and creds
+# Create the S3 Bucket and creds
```
'C' for creds

brian-armory commented on _spinnaker_install_admin_guides/install-on-aws.md:
```diff
@@ -123,156 +137,163 @@ Then, copy your kubeconfig file (this is typically located in `~/.kube/config) t
+10. Click "Create Policy"
 
-On the `Halyard machine`, choose a local working directory for Halyard.  In it, we will create two folders:
+# Stage files on the `Halyard machine`
```
not sure why Halyard machine is a codephrase (using the backticks)

brian-armory commented:
Woops meant to click request changes not approve.

aimeeu commented:
Thanks for reviewing this even though it's still a WIP! I didn't look closely at the entire [generated output](https://deploy-preview-740--stoic-hopper-290ccc.netlify.com/spinnaker-install-admin-guides/install-on-aws/), just the paragraph I added.  Netlify only builds PRs. I wanted to get the changes pushed because my Mac sometimes does weird things at night... like spontaneously rebooting or deciding to change my vertical monitor to horizontal and lose my 32" monitor altogether...

I literally moved one paragraph and added another. Looks like Atom's Markdown plugin formatted all the entire page.  So going forward, I'll fix everything on the page. Has to be done sometime...

aimeeu commented:
I discovered that I had installed tidy-markdown in Atom, and it was set to clean on save. I unchecked that box, so hopefully no more unintentional clean up of a file.

aimeeu commented:
Deploy preview:  https://deploy-preview-740--stoic-hopper-290ccc.netlify.com/spinnaker-install-admin-guides/install-on-aws/
